modules/cnf-creating-nrop-cr.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-creating-nrop-cr_{context}"]
+= Creating the NUMAResourcesOperator custom resource
+
+When you have installed the NUMA Resources Operator, then create the `NUMAResourcesOperator` custom resource (CR) that instructs the NUMA Resources Operator to install all the cluster infrastructure needed to support the NUMA-aware scheduler, including daemonsets and APIs.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NUMA Resources Operator.
+
+.Procedure
+
+. Save the following YAML in the `nrop.yaml` file:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1alpha1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+  - machineConfigPoolSelector:
+      matchLabels:
+        pools.operator.machineconfiguration.openshift.io/worker: ""
```

Comment:
let's add a note here that the worker nodes should be under this mcp label ("worker"), if the non-masters have a diiferent machine config pool selector it's fine but it must still match the configuration under ` machineConfigPoolSelector` in `nrop.yaml`.